### PR TITLE
feat: Add component monitoring

### PR DESCRIPTION
**Pull request by zhengjianhui**
Add component monitoring

### CODE REVIEW

File: apm/docs/docker-compose.md [hunk 1]
```diff
@@ -329,27 +329,46 @@ rule_files:
...
     static_configs:
     - targets: [node_exporter_ip]
   
   # 这里配置所有 muta 节点， ['node_id_1:8000', 'node_ip_2:8000', 'node_ip_3:8000']
-  # 端口请查看 muta chain.toml 配置下 [graphql] listening_address 参数
+  # Muta 应用的状态监控
   - job_name: 'muta_exporter'
     static_configs:
-    - targets: ['muta_exporter_ip']
+    - targets: ['192.168.20.211:8000']
```
Comment (homura) [inline apm/docs/docker-compose.md:359]: why hard code here?

File: apm/docs/metrics.md [hunk 2]
```diff
@@ -647,6 +625,93 @@ The value of the Metric variable up is zero
...
+#### Jaeger Agent Status
+- type: Jaeger
+- description: Jaeger agent status
+<details>
+<summary>Legende details</summary>
+
+##### active
+Number of Jaeger-agent services in up status
+```
+count(up{job="jaeger_agent"} == 1)
```
Comment (homura) [inline apm/docs/metrics.md:637]: too many panels can be a distraction, monitoring is about identifying risks, and we don't need to spend too much effort focusing on healthy nodes.

File: apm/docs/docker-compose.md (hunk 1 above)
Comment (zhengjianhui) [reply apm/docs/docker-compose.md:359]: Mistake. Here's an example.

File: apm/docs/metrics.md (hunk 2 above)
Comment (zhengjianhui) [reply apm/docs/metrics.md:637]: This is still important for checking the status of the components.

**Pull request merged**